bitnami/mariadb/templates/networkpolicy-egress.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if and .Values.networkPolicy.enabled (or .Values.networkPolicy.egressRules.denyConnectionsToExternal .Values.networkPolicy.egressRules.customRules) }}
+apiVersion: {{ include "common.capabilities.networkPolicy.apiVersion" . }}
+kind: NetworkPolicy
+metadata:
+  name: {{ printf "%s-egress" (include "common.names.fullname" .) }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  podSelector:
+    matchLabels:
+      {{- include "common.labels.standard" . | nindent 6 }}
```

Comment:
In Drupal we used
https://github.com/bitnami/charts/blob/ac6addfc22edfb912999e90d5a40e7198b26d3b0/bitnami/drupal/templates/networkpolicy-egress.yaml#L14-L16
Should we change it here or there?

Reply:
In drupal you have a `drupal` and a `mariadbp` pod that have different labels. Here in mariadb, the primary and secondary nodes have in common:
```
      app.kubernetes.io/name: mariadb
      helm.sh/chart: mariadb-9.8.0
      app.kubernetes.io/instance: mymariadb
      app.kubernetes.io/managed-by: Helm
```
That is also why the node type is included [here](https://github.com/bitnami/charts/blob/9a5d142aecf4682252488b3aa470fb89d0da39cc/bitnami/mariadb/templates/primary/statefulset.yaml#L7) or [here](https://github.com/bitnami/charts/pull/8079/files#diff-1aef2b0588aaa2c7cc6be8cf6417283b9dbb2eeb46a0326e29054c17143c992eR16)

Reply:
Ah got it, thanks!